mtp_api/apps/core/views.py

```diff
@@ -70,6 +70,9 @@ class AdminReportView(AdminViewMixin, FormView, metaclass=MediaDefiningClass):
     """
     form_class = NotImplemented
 
+    # test prisons that should not feature in any report
+    excluded_nomis_ids = {'ZCH'}
```

Comment:
I've seen this value 'ZCH' pop up a few times in other files, can it be extracted to a global setting (I see that's pretty much what you have done for this file...) for excluded NOMIS Ids or will it never change? Or is it a test prison as doesn't exist and just exists for testing nomis?

Reply:
Having looked, it's only in 3 places which doesn't make it essential for me yet.

Reply:
i think you're right. it would probably be a good idea to centralise this somehow. maybe on the Prison queryset/manager. it is in fact a test prison that exists in our production data, but does not in fact exist in NOMIS. in some situations we _might_ want to see it so it's not excluded super-globally.